addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -0,0 +1,54 @@
+import { merge } from '@ember/polyfills';
+import getElement from './-get-element';
+import triggerEvent from './trigger-event';
+import { KEYBOARD_EVENT_TYPES } from './fire-event';
+
+const DEFAULT_MODIFIERS = Object.freeze({
+  ctrlKey: false,
+  altKey: false,
+  shiftKey: false,
+  metaKey: false,
+});
+
+/**
+  @public
+  @param {String|HTMLElement} target
+  @param {'keydown' | 'keyup' | 'keypress'} eventType
+  @param {String} keyCode
+  @param {Object} [modifiers]
+  @param {Boolean} [modifiers.ctrlKey=false]
+  @param {Boolean} [modifiers.altKey=false]
+  @param {Boolean} [modifiers.shiftKey=false]
+  @param {Boolean} [modifiers.metaKey=false]
+  @return {Promise<void>}
+*/
+export default function triggerKeyEvent(target, eventType, keyCode, modifiers = DEFAULT_MODIFIERS) {
```

Comment:
When I initially copied `keyEvent`, I continued to use `keyCode` as the main way to set the key, but it seems to be [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) and the new alternatives are [KeyboardEvent.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) and [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key), specially the last one.

Perhaps we might want to start using it, or maybe assume that integers mean `keyCode` and strings mean `key`.

Reply:
Agreed, I'll try to review the various docs and update accordingly...

Reply:
Created https://github.com/emberjs/ember-test-helpers/issues/265 to track this.